hazelcast-sql-core/src/test/java/com/hazelcast/sql/misc/SqlNoDeserializationTest.java

```diff
@@ -99,7 +100,7 @@ public void after() {
     private static Config config() {
         Config config = smallInstanceConfig();
 
-        config.addMapConfig(new MapConfig(MAP_NAME).setInMemoryFormat(InMemoryFormat.BINARY));
+        config.addMapConfig(new MapConfig(MAP_NAME).setInMemoryFormat(InMemoryFormat.BINARY).setBackupCount(0));
```

Comment:
Why do we set backup count to zero here?

Reply:
A leftover. Removed.